### ISSUE TITLE
Merge into Documenter.jl

### ISSUE THREAD

**Issue by ParadaCarleton** [ali-ramadhan/DocumenterCitations.jl#66]
Hi, this looks like an amazing package you've got! Would you be willing to merge it into Documenter.jl, to make sure it can stay up-to-date and well-maintained into the future? Thank you!

**Comment by mortenpi** [comment]
> Would you be willing to merge it into Documenter.jl, to make sure it can stay up-to-date and well-maintained into the future?

Unfortunately, I am not sure I would accept it into Documenter, since I don't think I would have the bandwidth to maintain this functionality there. Generally, my vision for Documenter is that it contains the minimal set of functions necessary for the main Julia manual. Extra stuff should be done via plugins, and the effort in Documenter should go towards making the life of plugin writers easier.

That said, _if_ there is a desire to make this package a bit more centrally maintained, then it could always find a home in the [JuliaDocs org](https://github.com/JuliaDocs/) (also related: #64).

**Comment by navidcy** [comment]
I agree w @ParadaCarleton that this is a great package. Also agree with @mortenpi perhaps merging w Documenter is not the way forward. But I think moving into JuliaDocs might be great!

**Comment by goerz** [comment]
This project has now moved to JuliaDocs: https://github.com/JuliaDocs/DocumenterCitations.jl
